cigaradvisor/blocks/promotion-banner/promotion-banner.js

```diff
@@ -80,20 +82,7 @@ export default async function decorate(block) {
   }
   const promotion = activePromotions[Math.floor(Math.random() * activePromotions.length)];
 
-  // create anchor
-  const anchor = document.createElement('a');
-  block.appendChild(anchor);
-  anchor.setAttribute('href', promotion.url);
-  anchor.setAttribute('target', !isInternal(promotion.url) ? '_blank' : '_self');
-  anchor.setAttribute('title', promotion.alt);
-
-  // add image and CLS placeholder
-  const picture = createOptimizedPicture(promotion.image, promotion.alt || '');
-  anchor.appendChild(picture);
-  getImageDimensions(picture.querySelector('img')).then((img) => {
-    const ratio = (parseInt(img.height, 10) / parseInt(img.width, 10)) * 100;
-    // noinspection JSUnresolvedVariable
-    picture.style.paddingBottom = `${ratio}%`;
-    anchor.style.maxWidth = `${img.width}px`;
+  await loadPromotionContent(promotion.content).then((content) => {
+    if (content) block.parentElement.replaceWith(content);
```

Comment:
I'm somewhat torn on this. This replaces the wrapper with the block - removing the wrapper element from the original dom. I know i put this here, and it is a fix for making sure this CTA is the same height as the other column. but i wonder if there's a css-only way to fix this w/ layout handling.